mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -108,7 +109,6 @@ object Config {
    */
   case class Common(
       command: Option[String] = None,
-      inputfile: Option[File] = None, // TODO Move "inputfile" into an "Input" configuration group
```

Comment:
Moving this into the `Input` configuration group brings the `ApalacheConfig` structure closer into alignment with the option group structure.